virt/virt-2-5-release-notes.adoc

```diff
@@ -67,6 +67,10 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 * The Containerized Data Importer (CDI) can now import containerDisk storage volumes from the container image registry at a faster speed and allocate storage capacity more efficiently. CDI can pull a containerDisk image from the registry in about the same amount of time as it would take to import from an HTTP endpoint. You can import the disk into a PersistentVolumeClaim (PVC) equal in size to the disk image to use the underlying storage more efficiently.
 
 //CNV-7168 - DataVolume and storage diagnostics
+* It is now easier to diagnose and troubleshoot issues when preparing virtual machine (VM) disks that are managed by DataVolumes:
++
+** For asynchronous image upload, an error message is returned, before the connection is closed, if the size of the virtual disk image is larger than the size of the target DataVolume.
```

Comment:
Can we give a more precise description or a tiny comment/note of 'the size of the virtual disk image' because I found users put off or don't care about 'virtual size' (the size of the guest disk that VM can read) and 'disk size' (how much space the image file occupies on the host file system) for their first use. Take an QCOW2 image for example, here we should use 'virtual size'. So how about 'if the virtual size of the disk image'?
```
$ qemu-img info cirros-0.4.0-x86_64-disk.img
image: cirros-0.4.0-x86_64-disk.img
file format: qcow2
virtual size: 44M (46137344 bytes)
disk size: 13M
cluster_size: 65536
Format specific information:
    compat: 1.1
    lazy refcounts: false
    refcount bits: 16
    corrupt: false
```

Reply:
Changed to "virtual size of disk image"